### PR TITLE
Dev alpha

### PR DESCRIPTION
**Pull request by devinleighsmith**
This is a merge of dev into dev-alpha

71428c6 merge commit

9cd58f1 corrections

### CODE REVIEW

File: openshift/pipelines/Jenkinsfile.cicd [hunk 1]
```diff
@@ -22,9 +22,6 @@ pipeline {
   environment {
     CI = 'true'
 
-    // The name of your application
```
Comment (Fosol) [inline openshift/pipelines/Jenkinsfile.cicd:25]: I thought you just added this value, but it looks like you're removing it now

File: frontend/src/features/properties/components/forms/subforms/PidPinForm.tsx [hunk 2]
```diff
@@ -50,11 +50,10 @@ const PidPinForm: FunctionComponent<PidPinProps> = (props: PidPinProps) => {
   return (
     <>
       <Form.Row className="d-inline-flex flex-nowrap">
-        <Form.Label className="required">PID</Form.Label>
+        <Form.Label className="req">*</Form.Label>PID
```
Comment (Fosol) [inline frontend/src/features/properties/components/forms/subforms/PidPinForm.tsx:53]: Seems odd to have the label outside like this?

**Review by Fosol — APPROVED** [review]

File: frontend/src/features/properties/components/forms/subforms/PidPinForm.tsx (hunk 2 above)
Comment (devinleighsmith) [reply frontend/src/features/properties/components/forms/subforms/PidPinForm.tsx:53]: Agreed, but this is how it is in current dev-alpha

File: openshift/pipelines/Jenkinsfile.cicd (hunk 1 above)
Comment (devinleighsmith) [reply openshift/pipelines/Jenkinsfile.cicd:25]: I added the ability to have this value come from the env vars, not just hardcoded.

**Review by jdtoombs — APPROVED** [review]

**Pull request merged**